web/packages/shared/components/AwsLaunchButton/AwsLaunchButton.tsx

```diff
@@ -53,7 +53,7 @@ export class AwsLaunchButton extends React.Component<Props> {
       <>
         <ButtonBorder
           textTransform="none"
-          width="90px"
+          width="123px"
```

Comment:
This doesn't look good in Connect :(
It's much wider than other buttons.
<img width="620" alt="image" src="https://github.com/gravitational/teleport/assets/20583051/c9d02af4-3919-4857-a532-6a981e1942c3">

Reply:
The designs (for the web) have the buttons all the same width but I agree it looks bad here. Is there a way to get all the buttons in web/connect to be the same?

Reply:
I believe we only reuse the same button only for AWS resource. 
Perhaps I would be easier to only override the width for Connect.